GES-PersonalProject/Assets/Scripts/Hazard.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Hazard : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if(collision.gameObject.CompareTag("Player"))
+        {
+            Debug.Log("Player entered hazard.");
+            CharacterController player = collision.GetComponent<CharacterController>();
+            player.Respawn();
+        }
+        else
+        {
+            Debug.Log("Something other than the player entered the hazard.");
+        }
+    }
+
+}
```

Comment:
Good. Unless your game utilized the physics from the class tutorial video, that else statement wouldn't be needed.

Reply:
Looks like everything's in good order